src/panels/NightSkyPanel/tabs/LocationTab.tsx

```diff
@@ -97,69 +106,73 @@ export function LocationTab() {
       <Transition mounted={anchor?.identifier !== 'Earth'} transition={'fade'}>
         {(styles) => (
           <div style={styles}>
-            <Alert w={'100%'} title={'Warning!'} icon={<WarningIcon />}>
-              <Text>{`You are not on Earth, but on ${anchor?.name}.`}</Text>
+            <Alert
+              w={'100%'}
+              title={t('globe-location.warning.title')}
+              icon={<WarningIcon />}
+            >
+              <Text>
+                {t('globe-location.warning.description', { name: anchor?.name })}
```

Comment:
Shouldn't we always have an anchor node?  Even if it is the root, that should never be `null` at least

Reply:
We _should_ but our properties can be undefined until we've received them from OpenSpace